.github/workflows/consume-release.yml

```diff
@@ -3,16 +3,43 @@
 # a corresponding event handler to generate releases based on this signal
 
 on:
-  workflow_run:
-    workflows: [Release]
-    types: [completed]
+  workflow_dispatch:
+    inputs:
+      workflowRunId:
```

Comment:
Shouldn't we keep around `workflow_run`? Then we can automatically extract the `workflowRunId` from the event payload. It's also good to keep around the `workflow_dispatch` just in-case we need to do manual runs.

I think having the `workflow_run` trigger around is better in terms of semantics, since `workflow_dispatch` can be triggered from anywhere.